### ISSUE TITLE
Filter using "multi select" field

### ISSUE THREAD

**Issue by paulyili** [twentyhq/twenty#6562]
## The ability to filter using the Multi Select field

For example, if you describing businesses to the type of business they are, you could use the Multi Select field. 
- Supplier
- Maintenance
- Software
- Advertising
- Customer

In the instance a business is more than one, E.g Supplier & Software
you can filter all "Suppliers" to get a list of suppliers,

This would be a very convenient way to organise business.

**Comment by brendanlaschke** [comment]
duplicate of #5358 which is blocked by either the development in pg_graphql or a replacement for it ...

**Comment by FelixMalfait** [comment]
Yes it sucks but we don't have a short-term fix for that :-(
